docs/apis/cli.md

```diff
@@ -105,6 +105,10 @@ The command line can be used to
 
         ./bin/flink list -r
 
+-   List running Flink jobs inside Flink YARN session:
+
+        ./bin/flink list -m yarn-cluster -yid <yarnApplicationID> -r
```

Comment:
Why is the client trying to connect even though the application has been finished already?

```
[cloudera@quickstart build-target]$  ./bin/flink list -m yarn-cluster -yid application_1447844011707_0036 -r
2016-06-13 06:51:34,581 INFO  org.apache.hadoop.yarn.client.RMProxy                         - Connecting to ResourceManager at quickstart.cloudera/10.0.2.15:8032
2016-06-13 06:51:35,017 ERROR org.apache.flink.yarn.YarnClusterDescriptor                   - The application application_1447844011707_0036 doesn't run anymore. It has previously completed with final status: SUCCEEDED

------------------------------------------------------------
 The program finished with the following exception:

org.apache.flink.runtime.leaderretrieval.LeaderRetrievalException: Could not retrieve the leader gateway
	at org.apache.flink.runtime.util.LeaderRetrievalUtils.retrieveLeaderGateway(LeaderRetrievalUtils.java:127)
	at org.apache.flink.client.CliFrontend.getJobManagerGateway(CliFrontend.java:844)
	at org.apache.flink.client.CliFrontend.list(CliFrontend.java:378)
	at org.apache.flink.client.CliFrontend.parseParameters(CliFrontend.java:986)
	at org.apache.flink.client.CliFrontend.main(CliFrontend.java:1034)
Caused by: java.util.concurrent.TimeoutException: Futures timed out after [10000 milliseconds]
	at scala.concurrent.impl.Promise$DefaultPromise.ready(Promise.scala:219)
	at scala.concurrent.impl.Promise$DefaultPromise.result(Promise.scala:223)
	at scala.concurrent.Await$$anonfun$result$1.apply(package.scala:107)
	at scala.concurrent.BlockContext$DefaultBlockContext$.blockOn(BlockContext.scala:53)
	at scala.concurrent.Await$.result(package.scala:107)
	at scala.concurrent.Await.result(package.scala)
	at org.apache.flink.runtime.util.LeaderRetrievalUtils.retrieveLeaderGateway(LeaderRetrievalUtils.java:125)
	... 4 more
```